src/int_key.rs

```diff
@@ -51,6 +53,48 @@ macro_rules! cw_int_keys {
 
 cw_int_keys!(for i8, u8, i16, u16, i32, u32, i64, u64, i128, u128);
 
+macro_rules! cw_uint_std_keys {
+    (for $($t:ty),+) => {
+        $(impl IntKey for $t {
+            type Buf = [u8; mem::size_of::<$t>()];
+
+            #[inline]
+            fn to_cw_bytes(&self) -> Self::Buf {
+                self.to_be_bytes()
+            }
+
+            #[inline]
+            fn from_cw_bytes(bytes: Self::Buf) -> Self {
+                Self::new(IntKey::from_cw_bytes(bytes))
+            }
+        })*
+    }
+}
+
+cw_uint_std_keys!(for Uint64, Uint128);
+
+macro_rules! cw_int_std_keys {
+    (for $($t:ty),+) => {
+        $(impl IntKey for $t {
+            type Buf = [u8; mem::size_of::<$t>()];
+
+            #[inline]
+            fn to_cw_bytes(&self) -> Self::Buf {
+                let mut bytes = self.to_be_bytes();
+                bytes[0] ^= 0x80;
+                bytes
+            }
+
+            #[inline]
+            fn from_cw_bytes(bytes: Self::Buf) -> Self {
+                Self::new(IntKey::from_cw_bytes(bytes))
```

Comment:
Here I would go through `Self::from_be_bytes` for symmetry

Reply:
That doesn't seem to exist for some of the types.

Reply:
Oh boi, this is bad. Uint64 and Uint128 missing. Could you check with the team to get those added?

Reply:
Sure thing

Reply:
https://github.com/CosmWasm/cosmwasm/issues/2269